.travis.yml

```diff
@@ -2,18 +2,31 @@ language: php
 
 sudo: false
 
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
you should cache only the ``/files`` subfolder, to avoid invalidating the Travis cache on each builds because of the packagist metadata cache (builds on this repo are not happening often enough to benefit from the metadata cache anyway, as the cache will almost always be stale for next builds)